---
layout: page
title: CV
permalink: /cv/
---

## Work Experience

### now - 2022: Postdoctoral Research Associate
@ Big Data Institute, University of Oxford, Kelleher group
#### projects:
* Lead developer of genomic dataset and ancestry simulator *msprime*.
    * Efficient implementation of SMC(k) using search trees.
    * Implementation of background selection.
* Enabling the efficient computation of likelihoods for a broader class
of Ancestral Recombination Graphs.

2018 - 2022: PhD
@ University of Edinburgh, Lohse Lab

#### projects:
* thesis: The Laplace transform in population genetics: from theory to efficient algorithms.
* gIMble: modular likelihood optimisation toolkit for genome sequence data and
bioinformatics pipeline identifying barriers to gene flow (gIMble) ([paper](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1010999)).
* statistical method and associated workflow to perform genome scans for beneficial
mutations, outperforming existing approaches ([paper](https://academic.oup.com/genetics/article/219/2/iyab119/6337979)).
* agemo: a graph-based automatic differentiation toolkit speeding up existing inference methods 1000x ([paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010532)).

## Selection of publications

* Laetsch DR, **Bisschop G**, Martin SH, Aeschbacher S, Setter D, Lohse K (2023). Demographically explicit scans for barriers to gene flow using gIMble. PLoS Genetics, 19(10), e1010999. 
* **Bisschop G** (2022). Graph-based algorithms for Laplace transformed coalescence time distributions. PLOS Computational Biology, 18(9), e1010532.
* Baumdicker F, **Bisschop G**, ..., Kelleher J (2022). Efficient ancestry and mutation simulation with msprime 1.0. Genetics, 220(3), iyab229.
* **Bisschop G**, Lohse K, Setter D (2021). Sweeps in time: leveraging the joint distribution of branch lengths. Genetics, 219(2), iyab119.

See [google scholar](https://scholar.google.com/citations?user=SLLAzEUAAAAJ&hl=en&oi=ao) for a complete list.